### PR TITLE
Grey Plushie Get!

### PR DESCRIPTION
**Pull request by SweetBlueSylveon**
## What Does This PR Do
Adds a new Grey plushie to the game. This adds a new sprite to go with it, and adds it as a loadout option. The grey has the same cooldown effect as the vox plushie, but instead of going skree, it says "Greetings." in wingdings (this is why the code looks fucky on the line, as DM really dislikes wingdings. It shouldn't affect anything though.)
Upon touching water the plushie will switch sprites to a "singed" variant and yell audibly and in chat, before going onto an extended interaction cooldown of 30 seconds. The singed variant also has a separate hug interaction text, saying "Ow..." instead of "Greetings." in wingdings. Both of these hug interactions have a 5 second cooldown.

## Why It's Good For The Game
There's a vox plushie, and I think that a grey plushie would be good too.

## Images of changes
image
image
image

## Changelog
:cl:Phoaly
add: Added the grey plushie to the toys.dm and loadout.dm files.
add: Gave the grey plushie a water interaction that changes it's sprite and makes it yell. It also changes the hug interaction voiceline.
imageadd: Added the grey plushie sprite to toy.dmi.
/:cl:

### CODE REVIEW

**Comment by Ionward** [comment]
Looks good, and I like the wingdings idea too! I think it'd be great if they could somehow dissolve or burn when hit with water too, but that might be tricky to code, I'm just spitballing

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -995,6 +995,45 @@
...
+	var/hug_cooldown = FALSE //Defaults the plushie to being off coolodown. Sets the hug_cooldown var.
+	var/scream_cooldown = FALSE //Defaults the plushie to being off cooldown. Sets the scream_cooldown var.
+	var/singed = FALSE
+
+/obj/item/toy/plushie/greyplushie/water_act(volume, temperature, source, method = REAGENT_TOUCH) //If water touches the plushie the following code executes.
+	. = ..()
+	singed = TRUE
+	icon_state = "grey_singed"
+	item_state = "grey_singed"//If the plushie gets wet the sprite changes to a singed version.
+	desc = "A ruined plushie of a grey. It looks like someone ran it under some water."
```
[the remark this answers is not in the document]
Comment (SweetBlueSylveon) [reply code/game/objects/items/toys.dm]: Corrected, tested, and committed.

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -995,6 +995,47 @@
...
+	var/singed = FALSE
+
+/obj/item/toy/plushie/greyplushie/water_act(volume, temperature, source, method = REAGENT_TOUCH) //If water touches the plushie the following code executes.
+	. = ..()
+	singed = TRUE
+	if(!singed)
+		return
+	icon_state = "grey_singed"
+	item_state = "grey_singed"//If the plushie gets wet the sprite changes to a singed version.
+	desc = "A ruined plushie of a grey. It looks like someone ran it under some water."
```
Comment (SteelSlayer) [inline code/game/objects/items/toys.dm:1014]: Sorry, I should have been more clear here, this is what I meant:
```dm
if(!singed)
    singed = TRUE
    icon_state = "grey_singed"
	item_state = "grey_singed"//If the plushie gets wet the sprite changes to a singed version.
	desc = "A ruined plushie of a grey. It looks like someone ran it under some water."
```
This way, we only set the singed, icon and description vars when it's *not* already singed.

Comment (SweetBlueSylveon) [reply code/game/objects/items/toys.dm]: Latest commit adds this but with an early return. It should ensure this isn't an issue.

File: paradise.dme [hunk 3]
```diff
@@ -5,6 +5,220 @@
 // END_INTERNALS
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "config"
```
Comment (SteelSlayer) [inline paradise.dme:8]: It seems with your last commit, you accidentally added a bunch of stuff to the dme, and also made a change in the example config toml file.

Comment (SweetBlueSylveon) [reply paradise.dme:8]: Oh whoops, that's a testing change. Lemme fix that really quick. I have to revert them every time I push. I did it earlier with the "complicated" commit.

**Review by SteelSlayer — APPROVED** [review]

**Comment by dearmochi** [comment]
I like the idea of species specific plushies like, as you mentioned, the Vox plushie or the IPC one. The reference to Greys being weak to water is also a very nice touch IMO.

That being said, while the code looks fine, I can't help but feel the sprite is a bit rough and isn't quite consistent with the rest of the plushies. It may be my monitor, but your sprite looks "unlit" and as a result pops out rather inconveniently to the eye. I suggest you spend more time refining the sprite (or possibly starting from scratch) so that it is more in line with what we have right now. I wish I could provide more insight into this.

**Comment by SweetBlueSylveon** [comment]
I don't feel theres much much I can do to rectify a sprite without proper feedback on what's wrong with it. I'm sure it could use a touchup, but we've been through this sprite revision twice already. I'm no professional artist or anything, so I don't have much confidence in my capability to start again from scratch. The others seemed to really like the revised sprites you can see in our preview up in the first post CL, qnd I've remade it to specific criticisms given.
I get that you feel it might not fit up to the same standard as others, but from what I can see and gather, it's roughly in line with the Vox plushie in terms of shading and outlining. Aside from that, as I said in a previous comment, it's based on qnother grey plushie to some extent, and falls roughly in line with that one too.
Qgain I'm not saying that I'm unwilling to make changes, but I couldn't tell you what I would change without a proper criticism of whats wrong. Specifics on lighting, palette choice, lining, something to work with y'know? Your comment doesn't exactly give me much to go on.

**Comment by SweetBlueSylveon** [comment]
image
image
image
image
Here are some images of the newer sprites, including a comparison to the old version. If y'all like this, I'll throw the images into the CL.

**Review by dearmochi — APPROVED** [review]
Objection withdrawn after improvements were made. Well done!

**Comment by SweetBlueSylveon** [comment]
image
image
image
image

**Comment by hal9000PR** [comment]
grey plushie was gotten

**Pull request merged**